packages/react-components/src/icons/alert.tsx

```diff
@@ -31,8 +38,8 @@ const alert = (
                     <g id="Group-10">
                       <g id="Group-4"></g>
                       <rect
+                        fill={color || steel.rgb}
```

Comment:
Can we set this with CSS? What we normally do is move the fill up to <svg and then target it in CSS

Reply:
In this case, moving the fill to the <svg doesn't work, here's how it looks:
<img width="313" alt="Screenshot 2022-06-10 at 10 20 15" src="https://user-images.githubusercontent.com/87330794/173073620-148f26dd-702c-4489-a728-0e6b92c636a3.png">

And by the way, why did we choose to use the icons like this instead of using as proper customizable react components?

Reply:
I asked the same question when i joined but I think its because if you import them the other way they aren't put into the bundle by webpack rather loaded separately so for performance.